torchci/pages/tts/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -281,9 +297,16 @@ export default function Page() {
     { name: "stopTime", type: "string", value: stopTime },
     { name: "granularity", type: "string", value: granularity },
     { name: "percentile", type: "float", value: ttsPercentile },
-    { name: "branch", type: "string", value: "master" },
+    { name: "branch", type: "string", value: branch },
   ];
 
+  const checkboxRef = useCallback(() => {
```

Comment:
This callback eventually gets set to the ref? Is it really a ref? Don't we have to use createRef or useRef or something?

Reply:
I tried `useRef` but it didn't work or I used it incorrectly. What I try to do here is to have execute a callback to select the job once data from Rockset becomes available and. I just look around some more and [this](https://medium.com/welldone-software/usecallback-might-be-what-you-meant-by-useref-useeffect-773bc0278ae) seems to provide a high level explanation for that behavior